src/test/request-functions.ts

```diff
@@ -1,10 +1,11 @@
 import * as request from "supertest";
 
-export const mutationRequest = async (query, data) => {
+export const mutationRequest = async (query, data, token) => {
   const response = await request(`http://localhost:${process.env.PORT}/graphql`)
     .post("/")
-    .send({ query: query, variables: data })
-    .set("Accept", "application/json");
+    .send({ query, variables: data })
+    .set("Accept", "application/json")
+    .set("Authorization", token ? token : "");
```

Comment:
daria pra usar o nullish coalescing operator aqui, não?
```suggestion
    .set("Authorization", token ?? "");
```